packages/terra-navigation-side-menu/src/NavigationSideMenu.jsx

```diff
@@ -77,23 +86,34 @@ const propTypes = {
    * An optional toolbar to display below the side menu action header
    */
   toolbar: PropTypes.element,
+  /**
+   * Renders either Navigation Side Menu or Drill-IN
+   */
+  variant: PropTypes.oneOf(['nav-side-menu', 'drill-in']),
```

Comment:
Use constants like in [button](https://github.com/cerner/terra-core/blob/3f404c33a4092f4e43dfcdbd86531727dd757b1b/packages/terra-button/src/Button.jsx#L107)

Reply:
https://github.com/cerner/terra-framework/pull/2161/commits/066ee15428f028d455cea1e95f242f1cf9caef53